lib/compress/zstd_lazy.c

```diff
@@ -1212,7 +1256,7 @@ size_t ZSTD_RowFindBestMatch_generic (
         U32* const row = hashTable + relRow;
         BYTE* tagRow = (BYTE*)(tagTable + relRow);
         U32 const head = *tagRow & rowMask;
-        U32 matchBuffer[32 /* maximum nb entries per row */];
+        U32 matchBuffer[64 /* maximum nb entries per row */];
```

Comment:
It's a bit proper to use a `NAMED_CONSTANT` to ensure propagation.
Also, the name could be self-describing, removing the need to annotate it.